Assets/_Scripts/Actions/ActionBase.cs

```diff
@@ -38,7 +38,7 @@ public abstract class ActionBase : ScriptableObject {
     // TODO: update params
     public abstract void ExecuteAction(FightingEntity user, List<FightingEntity> targets);
 
-    public List<FightingEntity> GetTargets(FightingEntity user, List<int> targetIds){ 
+    public List<FightingEntity> GetPotentialTargets(FightingEntity user) {
```

Comment:
Is there an Action that could hit all? IE Earthquake from pokemon?

Reply:
We can add it to the GetPotentialTargets function later if there is.